docs/File-Formats.md

```diff
@@ -1248,3 +1250,6 @@ The cells contain the p-value for the GSVA score: A real number, between 0.0 and
 <tr><td>GO_GLUCURONATE_METABOLIC_PROCES</td><td>0.6621</td><td>0.0031</td><td>1.52e-9</td></tr>
 <tr><td>..</td><td></td><td></td><td></td></tr>
 </table>
+
+## Study Metadata file
+YAML or JSON file which contains extra information (metadata) about the cancer study. No compulsory fields are required for this file (free-form). To enable this feature, you need to add a line in the cancer study meta file with `metadata_filename:` followed the YAML/JSON file name. The information on the YAML or JSON file will be displayed in a table when mousing over a tag logo in the studies on the query page.
```

Comment:
JSON is also allowed? The validator only checks for correct YAML format, right? And The there is also only a unit test for YAML format. Could you add a test for JSON format then as well?

Reply:
Yes, you can use both YAML or JSON.

Reply:
> Technically YAML is a superset of JSON. This means that, in theory at least, a YAML parser can understand JSON, but not necessarily the other way around.

Which is the case here because the test I introduced checks if the metadata is not valid, and I had a hard time to find how to make it invalid 😄 But I'll introduce two tests that check that validating YAML and JSON formats for metadata do not report errors.

Reply:
I think it is confusing to call this `metadata_filename` since `meta_study.txt` is already a study metadata file. Maybe change the name slightly here to `extra_metadata_filename`?